atomate/vasp/firetasks/glue_tasks.py

```diff
@@ -202,6 +203,52 @@ def run_task(self, fw_spec):
         return FWAction(stored_data=stored_data)
 
 
+@explicit_serialize
+class GetInterpolatedPOSCAR(FiretaskBase):
+    """
+    Grabs CONTCARS from two previous calculations to create interpolated
```

Comment:
docs should be clearer about what the code does. The code is creating folders, writing POSCARs, copying CONTCARs, etc. You should write out clearly what the task is going to do.

Reply:
Done.